README.md

```diff
@@ -53,287 +53,6 @@ and you will find 3 available packages:
 
 then import `KarhooUISDK` wherever you want to access Karhoo services
 
-
```

Comment:
We agreed to remove also this very outdated visual demo from line 11:
<a href="https://gph.is/g/aQqR1p2" target="_blank">Visual Demo</a>